packages/client-sdk-nodejs/src/config/middleware/request-logging-middleware.ts

```diff
@@ -27,11 +25,12 @@ class ExperimentalLoggingMiddlewareRequestHandler
   }
   onRequestBody(request: MiddlewareMessage): Promise<MiddlewareMessage> {
     this.logger.debug(
-      'Logging middleware: request %s onRequestBody: request type: %s, request size: %s',
+      'Logging middleware: request %s onRequestBody: request type: %s, request size: %s, details: %s',
       this.requestId,
       // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      request.constructor.name,
```

Comment:
this used to just print `MiddlewareMessage` rather than the request type

Reply:
I think there are probably a few other places in the middleware where calling `constructorName()` will be more useful but haven't tracked them all down yet